ntbs-service/DataAccess/NotificationRepository.cs

```diff
@@ -50,6 +52,7 @@ public  IQueryable<Notification> GetDraftNotificationsIQueryable()
         public async Task AddNotificationAsync(Notification notification)
         {
             _context.Notification.Add(notification);
+            _context.AddAuditCustomField(CustomFields.AuditDetails, AuditType.Added);
```

Comment:
For all but add, we had a custom field - thought it would make sense to add this!